### PR TITLE
Duke Component

### PR DESCRIPTION
**Pull request by ddhenry95**

### CODE REVIEW

**Review by omonkulov — APPROVED** [review]
Looks good!

**Review by omonkulov — APPROVED** [review]

File: pages/duke.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import axios from "axios";
+const TEST_URL = "https://api.openweathermap.org/data/2.5/weather?q=LasVegas&appid=";
+
+export default function Sardorbek() {
+	const [temp, setTemp] = useState(null);
+
+	useEffect(() => {
+		axios.get(TEST_URL + process.env.NEXT_PUBLIC_OPEN_WEATHER_API_KEY).then((res) => {
+			setTemp(res.data.main.temp);
+            console.log(res.data);
```
Comment (cgm42) [inline pages/duke.js:12]: I think we should remove this console.log.

Comment (cgm42) [reply pages/duke.js:12]: Let's keep it for now and merge it first.

**Review by cgm42 — APPROVED** [review]
looks good

**Pull request merged**